commons/src/main/scala/com/lynbrookrobotics/potassium/commons/drivetrain/UnicycleDrive.scala

```diff
@@ -33,6 +33,7 @@ trait UnicycleHardware {
 
   val forwardPosition: Signal[Length]
   val turnPosition: Signal[Angle]
+  val position: Signal[Point]
```

Comment:
That's why I was pushing for only using relative positions in the tasks. Hardware traits should only contain signals that are pulled directly from sensors, and should not contain intermediate calculations. So you shouldn't be converting to a polling signal, and should just directly use relative positions (and maybe take the initial position as an argument to the task).